Src/Web/Web.Shared.Net.Tests/HandlerTelemetryProcessorTests.cs

```diff
@@ -103,5 +103,31 @@ public void ProcessReturnsIfSuccessfulRequestHandlerIsFiltered()
 
             Assert.AreEqual(0, spy.ReceivedCalls);
         }
+
+        [TestMethod]
```

Comment:
What about the processors? Where should I put the ResolvePlatformContext method?